common/src/app/common/types/container.cljc

```diff
@@ -166,13 +166,6 @@
     :else
     (get-instance-root objects (get objects (:parent-id shape)))))
 
-(defn get-copy-root
```

Comment:
This helper was really redundant, it does the same as get-component-shape and was used only in one place.